app/controllers/bestuurseenheid/reglementen/index.js

```diff
@@ -0,0 +1,48 @@
+import Controller from '@ember/controller';
+import { tracked } from '@glimmer/tracking';
+import { service } from '@ember/service';
+import { restartableTask } from 'ember-concurrency';
+
+const PAGE_SIZE = 10;
+
+export default class ReglementenIndexController extends Controller {
+  @service store;
+  @tracked from;
+  @tracked to;
+  @tracked page = 0;
+  @tracked bestuurseenheid;
+  @tracked model;
+  today = new Date().toISOString().substring(0, 10);
+  queryParams = ['page'];
+
+  fetchMeetings = restartableTask(async () => {
+    this.model = null;
```

Comment:
don't overwrite the model property on a controller...
https://guides.emberjs.com/release/routing/query-params/ please read this again

Reply:
I got this practice from here: https://github.com/lblod/frontend-gelinkt-notuleren-publicatie/blob/master/app/controllers/bestuurseenheid/index.js#L115 . I just wanted to stay consistent with other controllers. I will have a look now how we can do it better